docs/sites.yml

```
@@ -5136,3 +5136,14 @@
   built_by: Ariadne Digital
   built_by_url: https://www.ariadnedigital.it
   featured: false
+  - title: OneDeck
```

```suggestion
- title: OneDeck
```

Fixing the yaml formatting

---

Thanks @lannonbr!

---

If you agree with the fix, click that "commit suggestion" button